gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/LocalSnapshots.kt

```diff
@@ -179,7 +179,7 @@ abstract class LocalSnapshots : DefaultTask() {
   ) {
     ZipFile(apk).use { zip ->
       zip.entries().asSequence().forEach { entry ->
-        if (entry.name.endsWith(".dex")) {
+        if (entry.name.matches(Regex("^classes.*\\.dex"))) {
```

Comment:
nit: we could restrict this further to only match the digits like `classes8.dex`, not sure if there's another lib that would conflict with this naming.